specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/preview/2020-09-01-preview/examples/RoleListAddOns.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01-preview",
+    "subscriptionId": "4385cf00-2d3a-425a-832f-f4285b1c9dce",
+    "resourceGroupName": "GroupForEdgeAutomation",
+    "deviceName": "testedgedevice",
+    "roleName": "IoTRole1",
+    "accept-language": [
+      "en-US"
+    ]
```

Comment:
The parameters do not match the example responses.

Reply:
Changed